### PR TITLE
Update page metadata - Final PR

### PR DESCRIPTION
**Pull request by laurapacilio**
**Description**
Updating the page description metadata on the remaining top-level terraform.io pages, as part of an ongoing effort to update all of the doc page metadata. At this time, I am not editing page copy. This PR is only about adding/updating the page description metadata to improve SEO and the SERP descriptions that search engines display to users in search results. 

Requested review scope:
- [x] Content touched by the PR only (typos, clarifications, tips)
- [ ] Code test (command and code block changes)
- [ ] Flow and language near changes (new/rearranged steps)
- [ ] Review everything (rewrites, major changes)

Review urgency:
- [ ] ASAP (bug fixes, broken content, imminent releases)
- [ ] 3 days (small changes, easy reviews)
- [x] 1 week (default)
- [ ] Best effort (very non-urgent)

I have:
- [x] Verified that all status checks have passed
- [x] Verified that preview environment has successfully deployed.
- [x] Assigned an appropriate label (either Platform or a product name)
- [x] Requested at least one review

### CODE REVIEW

**Comment by laurapacilio** [comment]
Note - I fixed one legitimately broken link + anchor, but the rest do not appear to actually be broken. Either they are 302 Found or they are restricted from CURL calls (for example, the terraform cloud login page). So I think we can ignore that and merge anyway, once these are approved.

File: content/source/docs/enterprise/admin/index.html.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: "enterprise"
 page_title: "Administration - Terraform Enterprise"
+description: |-
+  Find documentation about the two types of Terraform Enterprise administration: Infrastructure and Application.
```
Comment (judithpatudith) [inline content/source/docs/enterprise/admin/index.html.md:5]:
```suggestion
  Learn how to administer the Terraform Enterprise application itself, and the infrastructure that it runs on.   
```

File: content/source/docs/glossary.html.md [hunk 2]
```diff
@@ -3,8 +3,7 @@ layout: terraform-glossary
 page_title: "Terraform Glossary"
 sidebar_current: "terraform-glossary"
 description: |-
-  Brief definitions of technical terms and jargon used in Terraform's
-  documentation and throughout the Terraform community.
+  Definitions for technical terms and jargon related to Terraform core, Terraform CLI, Terraform Cloud, and Terraform Enterprise.
```
Comment (judithpatudith) [inline content/source/docs/glossary.html.md:6]:
```suggestion
  Definitions for technical terms related to Terraform core, Terraform CLI, Terraform Cloud, and Terraform Enterprise. 
```

File: content/source/docs/registry/private.html.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: "registry"
 page_title: "Terraform Registry - Private Registry"
 sidebar_current: "docs-registry-private"
 description: |-
-  Terraform can load private modules from private registries via Terraform Cloud.
+  Terraform Cloud allows Terraform to load private modules from private registries.
```
Comment (judithpatudith) [inline content/source/docs/registry/private.html.md:6]: I might leave this as it was. It's a super subtle distinction, but Terraform inherently has the ability to load modules from private registries, and Terraform Cloud provides one such registry. The edit makes it sound like Terraform Cloud is adding something to Terraform binary's capabilities, but I think it actually complies with Terraform binary's standards. Hope that makes sense :P

**Review by judithpatudith — APPROVED** [review]
Looks great, thanks for doing these!

File: content/source/docs/cloud/agents/index.html.md [hunk 4]
```diff
@@ -25,9 +27,9 @@ Agents support Terraform versions 0.12 and above. Workspaces configured to use T
 
 ### Hardware Requirements
 
-The host running the agent will have varying resource requirements depending on the workspace. A host can be a dedicated or shared cloud instance, virtual machine, bare metal server, or a container. You should monitor and adjust memory, CPU, and disk space based on each workspace's usage and performance. The name of your instance type may vary depending on your deployment environment. 
+The host running the agent will have varying resource requirements depending on the workspace. A host can be a dedicated or shared cloud instance, virtual machine, bare metal server, or a container. You should monitor and adjust memory, CPU, and disk space based on each workspace's usage and performance. The name of your instance type may vary depending on your deployment environment.
 
-The specs below are provided as a reference. 
+The specs below are provided as a reference.
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply content/source/docs/cloud/agents/index.html.md:32]: yes! will fix!

[the remark this answers is not in the document]
Comment (laurapacilio) [reply content/source/docs/cloud/agents/index.html.md:32]: yes! I'll push up a fix!

File: content/source/docs/cloud/api/index.html.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
 ---
 page_title: "API Docs - Terraform Cloud and Terraform Enterprise"
 layout: "cloud"
+description: |-
+  Learn about authentication, available features, JSON API formatting, rate limiting, and other important information.
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply content/source/docs/cloud/api/index.html.md:5]: yup! I'll push up a change to do this!

File: content/source/docs/cloud/integrations/splunk/index.html.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 layout: "cloud"
 page_title: "Terraform Cloud for Splunk setup instructions"
 description: |-
-  HashiCorp Terraform Cloud customers can integrate with Splunk® using the official Terraform Cloud for Splunk app to understand Terraform Cloud operations
+  Splunk integration helps you identify the most active policies, significant changes in resource operations, and more.
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply content/source/docs/cloud/integrations/splunk/index.html.md:5]: yes! I'll push up a change that includes this!

File: content/source/docs/cloud/run/api.html.md [hunk 7]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: "cloud"
 page_title: "API-driven Runs - Runs - Terraform Cloud and Terraform Enterprise"
+description: |-
+  Your tool decides when configurations have changed and communicates with Terraform Cloud to upload configurations and trigger runs.
```
[the remark this answers is not in the document]
Comment (laurapacilio) [reply content/source/docs/cloud/run/api.html.md:5]: yes! it does - thank you for pointing that out, haha. I fixed it to read "External tools communicate with Terraform Cloud to upload configurations and trigger runs."  I think that's better!

File: content/source/docs/registry/private.html.md (hunk 3 above)
Comment (laurapacilio) [reply content/source/docs/registry/private.html.md:6]: it does! good catch - I'll change it back!

**Pull request merged**